packages/cli/src/commands/prisma.js

```diff
@@ -66,7 +66,7 @@ export const handler = async ({ _, $0, commands = [], ...options }) => {
   // Convert command and options into a string that's run via execa
   let args = commands
   for (const [name, value] of Object.entries(options)) {
-    args.push(`--${name}`)
+    args.push(name.length > 1 ? `--${name}`: `-${name}`)
```

Comment:
I suggest this:

```
  // Convert command and options into a string that's run via execa
  const args = commands
  for (const [name, value] of Object.entries(options)) {
    // Allow both long and short form commands, e.g. --name and -n
    args.push(name.length > 1 ? `--${name}` : `-${name}`)
    if (typeof value !== 'boolean') {
      args.push(value)
    }
  }
```  

1. makes `args` a constant because we're not reassigning it
2. Adds a comment about what's going on
3. Fixes the lint error (space)

Reply:
@Tobbe just applied!